README.md

```diff
@@ -101,6 +101,32 @@ hvac_zones = orm.session.query(Location)\
 print(f"Model has {len(hvac_zones)} HVAC Zones")
 ```
 
+## Validate with Shape Constraint Language
+
+The module utilizes the [pySHACL](https://github.com/RDFLib/pySHACL) package to validate a building ontology
+against the Brick Schema, its default constraints (shapes) and user provided shapes.
+
+```python
+from brickschema.validate import Validator
+from rdflib import Graph
+
+dataG = Graph()
+dataG.parse('myBuilding.ttl', format='turtle')
+shapeG = Graph()
+shapeG.parse('extraShapes.ttl', format='turtle')
+v = Validator()
+result = v.validate(dataG, shacl_graphs=[shapeG])
+# if result.conforms == True, textOutput is "Validation Report\nConforms: True"
+print(result.textOutput)
+```
```

Comment:
You had some nice human-readable output you developed, right? Is there a good way of getting that output from the result object? I think we should bring that out in the documentation: can you add a line or two to the example code that shows how I can get the nice output?

Reply:
The argumented output is in result.textOutput which is the original output of pyshacl.validate appended with our "extra info".  There is sample output in the rst file for brick_validate.  Do you want the sample output to be shown in README, too?

Reply:
If it's included in `textOutput` then you don't need to include the sample output itself, but can you add a line or two to the documentation to indicate that this  includes a detailed validation report?

```python
 # if result.conforms == True, textOutput is "Validation Report\nConforms: True"
# if there is a validation error, textOutput will include a detailed report
```

It's worth including a line pointing users to the `validationGraphs` as well, just in case they are familiar with SHACL

Reply:
done